packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -134,6 +135,12 @@ class Dismissible extends StatefulWidget {
   /// [direction] property.
   final Map<DismissDirection, double> dismissThresholds;
 
+  final Duration dismissDuration;
+  /// defines the duration for card to dismiss and come to original position
```

Comment:
Dart docs above the member being documented and in complete sentences (capitalize and period).